.dangerfile/readme_tests.rb

```diff
@@ -424,7 +424,6 @@ def readme_invalid_credentials?(file)
           if !line.split("  - ")[1].match?(flexera_perm_tester)
             fail_message += "Line #{line_number.to_s}: Flexera permission list item formatted incorrectly. Please make sure all list items are formatted like the following examples:\n\n"
             fail_message += "```  - `billing_center_viewer`*```\n"
```

Comment:
If the below example is being removed, either replace it with a new one, or add a second \n to this line. Otherwise, the spacing between errors raised by Dangerfile will be wrong.

Reply:
Applied at [aa8bf5b](https://github.com/flexera-public/policy_templates/pull/2262/commits/aa8bf5b105455edae7bc597bb510b98a94b7725f)